.circleci/config.yml

```diff
@@ -11,12 +27,52 @@ jobs:
 
     steps:
       - checkout
-      - run:
-          name: Install dependencies
-          command: |
-            gem install bundler
-            bundle check || bundle install --retry=3 --path vendor/bundle
+      - bundle-install
       - run:
           name: Run Tests
           command: |
             bundle exec rspec spec
+
+  deploy:
+    docker:
+      - image: cimg/ruby:2.7.2
+    working_directory: ~/identity-fake-server
+    parameters:
+      space:
+        type: string
+    steps:
+      - checkout
+      - bundle-install
+      - build-release
+      - run:
+          name: Install cf cli
+          command: |
+            curl -v -L -o cf-cli_amd64.deb 'https://cli.run.pivotal.io/stable?release=debian64&source=github'
+            sudo dpkg -i cf-cli_amd64.deb
+            cf -v
+
+      - run:
+          name: login to cloud.gov
+          command: |
+            cf login -a https://api.fr.cloud.gov -u "4a945e17-3364-4ff2-9c35-2078cd2a94b4" -p $CF_PASS -o "gsa-login-prototyping" -s "<< parameters.space >>"
+
+      - run:
+          name: deploy to cloud.gov
+          command: |
+            cf push identity-fake-server
+
+workflows:
+  version: 2
+  build_test_deploy:
+    jobs:
+      - build
+      - deploy:
+          requires:
+            - build
+          filters:
+            branches:
+              only: main
+          matrix:
+            parameters:
+              space:
+                - pt
```

Comment:
might be overkill to have a matrix of 1 element, but seemed easier to just copy the structure for now